griptape/loaders/text_loader.py

```diff
@@ -41,7 +42,7 @@ def text_to_artifacts(self, text: str | Path) -> list[TextArtifact]:
         artifacts = []
 
         if isinstance(text, Path):
-            with open(text, "r") as file:
+            with open(text, "r", encoding=self.encoding) as file:
```

Comment:
Should we also integrate it into `TextArtifact` that we instantiate below? Similar to how we do it `BlobArtifact`.

Reply:
Would the field on `TextArtifact` just be for reference? Or would it enable some functionality like `to_bytes`?

Reply:
I think just for reference for now.